packages/api/src/schemas/envConfig.json

```diff
@@ -6,6 +6,39 @@
     "$schema": {
       "type": "string"
     },
+    "auth": {
```

Comment:
@adashen @KennethBWSong , please help review the schema for `auth`, which is for existing AAD app customization according to previous discussion.